components/mpas-framework/src/framework/framework.cmake

```diff
@@ -32,4 +32,5 @@ list(APPEND COMMON_RAW_SOURCES
   framework/regex_matching.c
   framework/mpas_field_accessor.F
   framework/mpas_log.F
+  framework/mpas_globalSumMod.F
```

Comment:
I think this should be updated with the new filename: `framework/mpas_global_sum_mod.F`. That probably explains the CI errors.

Reply:
But without the mod, because no other modules have that. Just `framework/mpas_global_sum.F`

Reply:
As mentioned above, the _mod in this case is necessary to distinguish it from the function name. And the filename should reflect the module name for automated dependency generation. I'll change the cmake file.